lstchain/reco/dl1_to_dl2.py

```diff
@@ -153,15 +151,15 @@ def train_disp_sign(train, custom_config={}, predict_feature='disp_sign'):
     """
 
     config = replace_config(standard_config, custom_config)
-    classification_args = config['random_forest_classifier_args']
-    features = config["classification_features"]
-    model = RandomForestClassifier
+    regression_args = config['random_forest_regressor_args']
+    features = config['regression_features']
+    model = RandomForestRegressor
```

Comment:
Why are you using a regressor for the sign?

Reply:
I think the code was correct before

Reply:
haha, because I didn't think

Reply:
will change that, thanks